MRFIXIT: when kodi exits, the display is sent a CEC code which usually turns off the display. This isn't ideal in batocera - unless it's a power-off event. A reboot CEC doesn't disable the monitor, so use that unless it's a power off event.
diff --git a/xbmc/application/Application.cpp b/xbmc/application/Application.cpp
index ad22f17..fec9775 100644
--- a/xbmc/application/Application.cpp
+++ b/xbmc/application/Application.cpp
@@ -2661,7 +2661,13 @@ void CApplication::Stop(int exitCode)
     m_frameMoveGuard.unlock();
 
     CVariant vExitCode(CVariant::VariantTypeObject);
-    vExitCode["exitcode"] = exitCode;
+ 
+    // Passing a CEC exit code usually powers off display - avoid that by pretending it's a reboot unless it's a power-off
+    if (exitCode == EXITCODE_POWERDOWN)
+      vExitCode["exitcode"] = exitCode;
+    else
+      vExitCode["exitcode"] = EXITCODE_REBOOT;
+ 
     CServiceBroker::GetAnnouncementManager()->Announce(ANNOUNCEMENT::System, "OnQuit", vExitCode);
 
     // Abort any active screensaver
